test/e2e/features/story_microshift.feature

```diff
@@ -35,4 +33,4 @@ Feature: Microshift test stories
 		And with up to "4" retries with wait period of "1m" http response from "http://httpd-example-testproj.apps.crc.testing" has status code "200"
 		Then executing "curl -s http://httpd-example-testproj.apps.crc.testing" succeeds
 		And stdout should contain "Hello CRC!"
-		Then with up to "4" retries with wait period of "1m" http response from "http://httpd-example-testproj.apps.crc.testing" has status code "200"
\ No newline at end of file
+		Then with up to "4" retries with wait period of "1m" http response from "http://httpd-example-testproj.apps.crc.testing" has status code "200"
```

Comment:
Is this change just adding new line?

Reply:
It’s my editor doing this automatically, but yes, it ends the file with a new line.